src/components/tutor-schedule/TutorSchedule.styles.ts

```diff
@@ -0,0 +1,89 @@
+export const styles = {
```

Comment:
ALL styles wariables should be in camelCase, like in add styles files along thr app